src/styles/_Base.scss

```diff
@@ -56,6 +56,10 @@ ul {
   list-style: circle inside;
 }
 
+dt {
+  font-weight: 700;
```

Comment:
Instead of hardcoding 700 here, you should use a variable `$font-weight-extra-bold`